pom.xml

```diff
@@ -70,6 +70,12 @@
         </developer>
     </developers>
     <profiles>
+        <profile>
+            <id>java-8-api</id>
```

Comment:
yes I think we can do this since I am removing the manual passing of these flags. Both maven-compiler-plugin and scala-maven-plugin automatically resolve the conflict giving precedence to release.

Reply:
removed this profile and documented how to specify flags if needed on the command line